docs/CONTRIBUTING.md

```diff
@@ -79,15 +79,25 @@ Available recipes:
 ## Deploying
 
 A reminder for the maintainers on how to deploy.
-Make sure the main branch is checked out and all desired changes
-are merged. Then run:
+1) Make sure all your changes are committed and merged into main.
+2) Make sure branch is clean:
+    ```bash
+    git checkout main
+    git stash
+    git pull
+    ```
+3) Create tag and push new version to GitHub like so:
+    ```bash
+    just tag-for-release "vX.Y.Z"
+    just release
```

Comment:
sanity check: is this accurate currently?

Reply:
Unless something has changed from my underlying cookiecutter, that is how I do it!

https://github.com/bioio-devs/bioio/blob/main/Justfile#L65

Reply:
Yep 👍  this is accurate